integration-tests/e2e-tests/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "homepage": "https://github.com/input-output-hk/atala-prism-wallet-sdk-ts-e2e",
   "dependencies": {
-    "@atala/prism-wallet-sdk": "../../",
```

Comment:
can you undo those changes?
Also in `yarn.lock` and `package-lock.json`

Reply:
We can review this later its not something we introduced in this PR @FabioPinheiro I'm going to resolve the comment.

The slash adds no difference.